indicators/NO_BFLY_001/R/NO_BFLY_001.qmd

```diff
@@ -0,0 +1,867 @@
+---
+title: "Butterflies"
+subtitle: |
+  [NO_BFLY_001] (grasslands) \
+  [NO_BFLY_002] (forests)
+format: 
+  html:
+    embed-resources: true
+    code-fold: true
+    toc: true
+    toc-title: Contents
+    toc-depth: 3
+    smooth-scroll: true
+execute: 
+  cache: true
+author:
+  - name: Jens Åström
+    email: jens.astrom@nina.no  # Enter email
+    affiliations:
+      - id: myID
+        name: Norwegian Institute for Nature Research
+  - name: Anders L. Kolstad
+    affiliations:
+      - ref: myID     
+date: January 14, 2025 # Enter date 
+callout-icon: false
+lightbox: true
+css: ../../../style.css
+code-links:
+      - text: Add a review
+        icon: github
+        href: https://github.com/NINAnor/ecRxiv
+bibliography: references.bib
+---
+
+<!--# This is a template for how to document the indicator analyses. Make sure also to not change the order, or modify, the headers, unless you really need to. This is because it easier to read if all the indicators are presented using the same layout. If there is one header where you don't have anything to write, just leave the header as is, and don't write anything below it. If you are providing code, be careful to annotate and comment on every step in the analysis. Before starting it is recommended to fill in as much as you can in the metadata file. This file will populate the initial table in your output.-->
+
+<!--# Load all you dependencies here -->
+
+```{r setup}
+#| include: false
+library(knitr)
+library(tidyverse)
+library(kableExtra)
+library(here)
+library(bombLepiSurv)
+library(tmap)
+library(sf)
+library(NinaR)
+library(CursoRopen)
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r source}
+#| echo: false
+source(here::here("_common.R"))
+```
+
+```{r}
+#| echo: false
+meta <- readxl::read_xlsx("../metadata.xlsx")
+st <- meta |>
+  filter(Variable == "status") |>
+  pull(Value)
+version <- meta |>
+  filter(Variable == "Version") |>
+  pull(Value)
+auth <- meta |>
+  filter(Variable == "authors") |>
+  pull(Value)
+year <- meta |>
+  filter(Variable == "yearAdded") |>
+  pull(Value)
+id <- meta |>
+  filter(Variable == "indicatorID") |>
+  pull(Value)
+name <- meta |>
+  filter(Variable == "indicatorName") |>
+  pull(Value)
+url <- meta |>
+  filter(Variable == "url") |>
+  pull(Value)
+
```

Comment:
the url is ...001 but the version is now 002. Should the url change or is this ok?

Reply:
Fixed